Functions/Example/FirebaseFunctions.xcodeproj/project.pbxproj

```diff
@@ -580,7 +604,13 @@
 					"DEBUG=1",
 					"$(inherited)",
 				);
-				HEADER_SEARCH_PATHS = "$(SRCROOT)/../FirebaseFunctions";
+				HEADER_SEARCH_PATHS = (
+					"$(inherited)",
+					"\"${PODS_ROOT}/Headers/Public\"",
+					"\"${PODS_ROOT}/Headers/Public/FirebaseAnalyticsInterop\"",
+					"\"${PODS_ROOT}/Headers/Public/FirebaseAuthInterop\"",
+					"\"$(SRCROOT)/../FirebaseFunctions\"/**",
```

Comment:
Thanks for addressing the Xcode issue. Recursive search paths are maybe ok for tests only, but would be nice to avoid  - they can hide issues and cause tricky bugs.